Lombiq.Hosting.Azure.ApplicationInsights.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -0,0 +1,23 @@
+using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Services;
+using OpenQA.Selenium;
+using Shouldly;
+
+namespace Lombiq.Hosting.Azure.ApplicationInsights.Tests.UI.Extensions;
+
+public static class TestCaseUITestContextExtensions
+{
+    /// <summary>
+    /// Tests Application Insights tracking during offline operation. Use <see
+    /// cref="OrchardCoreUITestExecutorConfigurationExtensions.EnableApplicationInsightsOfflineOperation"/> to switch
+    /// the module into offline mode.
```

Comment:
Shouldn't we verify this during runtime explicitly? For example add a `internal static IsOfflineOperationEnabled { get; private set; }` to `OrchardCoreUITestExecutorConfigurationExtensions`. Then `EnableApplicationInsightsOfflineOperation` sets it to `true`. This method checks and throws an exception with instructions if it's `false`. I see there is a message in the `ShouldBe`, but that's generic and doesn't tell about `EnableApplicationInsightsOfflineOperation`.

Reply:
Yes, good point.